Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -282,4 +282,24 @@ lemma normal_iff_forall_map_eq : Normal F K ↔ ∀ σ : L →ₐ[F] L, K.map σ
 lemma normal_iff_forall_map_eq' : Normal F K ↔ ∀ σ : L ≃ₐ[F] L, K.map ↑σ = K :=
 ⟨fun h σ ↦ normal_iff_forall_map_eq.1 h σ, fun h ↦ normal_iff_forall_map_le'.2 (fun σ ↦ (h σ).le)⟩
 
+instance instSMulMemClass : SMulMemClass (IntermediateField F L) F L :=
```

Comment:
Is this instance needed? At least in this file, it can be commented out.

Reply:
It is only used in #16993, any suggestions about where to move it? Right before where it is used?